play/src/front/Connection/ConnectionManager.ts

```diff
@@ -576,6 +576,21 @@ class ConnectionManager {
         }
     }
 
+    async searchMembers(searchText: string): Promise<MemberData[]> {
+        const playUri = this.currentRoom?.key;
+        if (playUri == undefined) {
+            throw new Error("playUri is undefined");
+        }
+
+        const response = await axiosToPusher.get<MemberData[]>("members", {
+            params: { playUri, searchText },
+            headers: {
+                Authorization: this.authToken,
+            },
+        });
+        return response.data ? response.data : [];
```

Comment:
I just remembered we have a notion of "query" to Pusher that allows us to pass messages to the Pusher in a type-safe way via Websockets.
Not sure we need to do it here, but some other time, we could use those queries instead of adding a new HTTP endpoint (that needs to be properly secured)